ropstenpeers.txt

```diff
@@ -1,2 +0,0 @@
-enode://30b7ab30a01c124a6cceca36863ece12c4f5fa68e3ba9b0b51407ccc002eeed3b3102d20a88f1c1d3c3154e2449317b8ef95090e77b312d5cc39354f86d5d606@52.176.7.10:30303
```

Comment:
This is just for my own benefit but what was this file for and why don't we need it anymore?